src/frontend/apps/e2e/__tests__/app-desk/mail-domains-create.spec.ts

```diff
@@ -0,0 +1,274 @@
+import { expect, test } from '@playwright/test';
+import { MailDomain } from 'app-desk/src/features/mail-domains';
+
+import { keyCloakSignIn } from './common';
+
+const currentDateIso = new Date().toISOString();
+const mailDomainsFixtures: MailDomain[] = [
+  {
+    name: 'local.fr',
+    id: '456ac6ca-0402-4615-8005-69bc1efde43f',
+    created_at: currentDateIso,
+    updated_at: currentDateIso,
+    slug: 'localfr',
+    abilities: {
+      get: true,
+      patch: true,
+      put: true,
+      post: true,
+      delete: true,
+      manage_accesses: true,
+    },
+  },
+];
+
+const mailDomainFrFixture = {
+  name: 'domain.fr',
+  id: 'dd92f0d0-4a9c-439c-91f4-08d28d480a3a',
+  created_at: currentDateIso,
+  updated_at: currentDateIso,
+  slug: 'domainfr',
+  abilities: {
+    get: true,
+    patch: true,
+    put: true,
+    post: true,
+    delete: true,
+    manage_accesses: true,
+  },
+};
+
```

Comment:
You don't need to mock anymore. Better to do all your checks without mocking, like that you test that the backend works as well as expected.